packages/design-system/src/components/web-components/ds-tooltip/ds-tooltip.tsx

```diff
@@ -0,0 +1,106 @@
+import { define } from '../preactement/define';
+import { Tooltip, TooltipProps } from '../../Tooltip';
+import { Placement } from '@popperjs/core';
+import { parseBooleanAttr } from '../wrapperUtils';
+
+const attributes = [
+  'active-class-name',
+  'class-name',
+  'close-button-label',
+  'component',
+  'content-heading',
+  'dialog',
+  'interactive-border',
+  'inversed',
+  'max-width',
+  'placement',
+  'root-id',
+  'show-close-button',
+  'title',
+  'transition-duration',
+  'trigger-aria-label',
+  'z-index',
+] as const;
+
+/* eslint-disable @typescript-eslint/no-namespace */
+declare global {
+  namespace JSX {
+    interface IntrinsicElements {
+      'ds-tooltip': React.DetailedHTMLProps<
+        React.HTMLAttributes<HTMLElement> & {
+          [K in (typeof attributes)[number]]?: string;
+        },
+        HTMLElement
+      >;
+    }
+  }
+}
+/* eslint-enable */
+
+interface WrapperProps
+  extends Omit<
+    TooltipProps,
+    | 'ariaLabel'
+    | 'contentHeading'
+    | 'dialog'
+    | 'inversed'
+    | 'showCloseButton'
+    | 'offset'
+    | 'placement'
+    | 'title'
+  > {
+  contentHeading?: string | TooltipProps['contentHeading'];
+  dialog?: string;
+  inversed?: string;
+  showCloseButton?: string;
+  placement?: string;
+  rootId?: string;
+  title: string | TooltipProps['title'];
```

Comment:
Same idea as above. Do you have a story to demonstrate that it's possible to pass in a ReactNode (e.g. what TooltipProps['title'] implies here)?